sys/src/cmd/paqfs/paqfs.c

```diff
@@ -261,14 +261,14 @@ main(int argc, char *argv[])
 		break;
 	default:
 		close(pfd[1]);	/* don't deadlock if child fails */
-		if(mnt && mount(pfd[0], -1, mntpoint, MREPL|MCREATE, "") < 0)
+		if(mnt && (mount(pfd[0], -1, mntpoint, MREPL|MCREATE, "", 'M') < 0))
```

Comment:
What's the 'M' for here?

Reply:
It's not in man, that flag was added by Ron to mount at early days (I think after an agreement with John and Aki, they were the gurus). 
```
commit 72777a1fab7b33c26bb4d9dccff934d4f213303e
Author: Ronald G. Minnich <rminnich@gmail.com>
Date:   Thu Aug 20 09:49:28 2015 -0700

    Expose the 'dc' parameter to user programs by extending the mount call.
    
    The mount call has another parameter, an int, which is the device character
    used to pick a mount device. This works for mount and ramfs,
    but it would be nice if someone else could verify it.
    
    Now, go forth, and write new devmnt devices!
    
    Change-Id: I765dd6830f5a26d8a96d5e568fe61871cf7e402c
    Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>
```
Usually, #M mnt is the only one used, that's why 99,9% of mount() calls you can see in src/cmd have 'M', but Ron made #N mntn for showing how to make more devmnt devices. The main purpose was making devmnt's for new drivers or fs that couldn't be necessarily 9P or that could have special requirements, like ZFS or ext4 as we thought those days.